i18n/README.jp.md

```diff
@@ -9,12 +9,12 @@
 [Supabase](https://supabase.io)は、オープンソースのFirebase代替製品です。エンタープライズグレードのオープンソースツールを使って、Firebaseの機能を構築しています。
 
 - [x] ホスティングされたPostgresデータベース
-- [x] リアルタイムサブスクリプション機能
+- [x] リアルタイムサブスクリプション
 - [x] 認証・認可
 - [x] APIを自動生成
 - [x] ダッシュボード
 - [x] ストレージ
-- [ ] ファンクションズ (近日公開)
+- [ ] 関数 (近日公開)
```

Comment:
Thanks for your suggestion! This one was hard to choose the correct word. I was thinking since cloud functions on GCP is called `Cloud Functions` in Japanese, do you think it would be better if we just call it `Functions` here as well?

Reply:
I guess AWS does call it Lamda関数, so 関数 might be good. I really struggles to decide whether I should keep these names all English or use katakana here 😂